src/app/api/aduser/api/v1/user/route.js

```diff
@@ -13,10 +13,17 @@ export async function GET(request) {
     const res = await fetch(userUrl, {
         method: "GET",
         headers: createAuthorizationAndContentTypeHeaders(token)
+    }).catch(err => {
+        logger.error(`GET user fetch feilet '${err}'`);
+        return new Response("Fetch feilet", {
+            status: 500
+        })
     });
 
     if (!res.ok) {
-        return new Response(res.body, {
+        const text = await res.text();
+        logger.error(`GET user feilet status: ${res.status} text: ${text}`);
```

Comment:
Vil ikke denne logge feil hver gang man da får 404?

Reply:
Jo - godt poeng! 😆 Jeg legger inn et unntak for 404